in_text/text_data_release.yml

```diff
@@ -140,23 +140,23 @@ entities:
   -
     data-name: 02_munge/src/fill_discharge_prms.py
     data-description: >-
-      XX
+      A python script retrieving prms preditions of discharge from science base data release and fills nan gaps. 
   -
     data-name: 02_munge/src/munge_noaa_nerrs.py
     data-description: >-
-      XX
+      A python script containing functions to process NOAA NERRS data.
   -
     data-name: 02_munge/src/munge_noaa_nos.py
     data-description: >-
-      XX
+      A python script containing functions to process NOAA NOS data.
   -
     data-name: 02_munge/src/munge_usgs_nwis.py
     data-description: >-
-      XX
+      A python script containing functions to process USGS NWIS data.
   -
     data-name: 02_munge/src/salt_front_record.R
     data-description: >-
-      XX
+      A R script containing functions to process daily and weekly salt front calculations from a historic record. 
```

Comment:
```suggestion
      An R script that renames columns and changes data types for daily and weekly values of salt front location (source: XX). 
```

Reply:
@galengorski - can you confirm this is an accurate description of all that is happening in this script? We will need to updated the XX with the link to the DRBC data once we get it (that's what this is right?)